docs/basic-features/image-optimization.md

```diff
@@ -51,6 +50,39 @@ function Home() {
 export default Home
 ```
 
+## Image Imports
+
+You can `import` images that live in your project. (Note that `require` is not supported—only `import`.)
+
+With direct `import`s, `width`, `height`, and `blurDataURL` will be automatically provided to the image component. Alt text is still needed separately.
```

Comment:
Let's call it "static imports" instead of "direct imports" (I don't think we use direct anywhere else)

Reply:
Good flag!
```suggestion
With static `import`s, `width`, `height`, and `blurDataURL` will be automatically provided to the image component. Alt text is still needed separately.
```